files/zh-cn/learn/forms/sending_and_retrieving_form_data/index.md

```diff
@@ -58,25 +58,25 @@ web 基于非常基本的客户端/服务器体系结构，可以总结如下:
 在这个例子中，数据被发送到一个绝对 URL —— `http://foo.com`：
 
 ```html
-<form action="http://foo.com">
+<form action="http://foo.com"></form>
```

Comment:
coud we also add `…`

![image](https://github.com/mdn/translated-content/assets/15844309/675fc6e7-ac96-4dd8-a75d-0aafe06c3756)

```suggestion
<form action="http://foo.com">…</form>
```